bootloaders/riotboot/Makefile

```diff
@@ -0,0 +1,25 @@
+# Default RIOT bootloader
+APPLICATION = riotboot
+
+# Default testing board
+BOARD ?= samr21-xpro
+
+# Select the boards with riotboot feature
+FEATURES_REQUIRED += riotboot
+
+# Disable unused modules
+CFLAGS += -DNDEBUG -DLOG_LEVEL=LOG_NONE
```

Comment:
Shouldn't this be conditional on DEBUG or DEVELHELP or something like that?

Reply:
In this case this is used intentionally to reduce at its maximum the size of the bootloader. Otherwise instances of the debugging module might be activated and thus making use of the libc, which increases the size of the bootloader by several kilobytes.

Activating them under other conditions implies to increase the size of the bootloader, which is not done automatically, and requires several modifications to the current approach.